src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -205,6 +218,90 @@ public WindowState WindowState
             }
         }
 
+        private void MarkFullscreen(bool fullscreen)
+        {
+            //if (!task_bar_list_)
+            //{
+            //    HRESULT hr =
+
+
+            //        ::CoCreateInstance(CLSID_TaskbarList, nullptr, CLSCTX_INPROC_SERVER,
+            //                           IID_PPV_ARGS(&task_bar_list_));
+            //    if (SUCCEEDED(hr) && FAILED(task_bar_list_->HrInit()))
+            //        task_bar_list_ = nullptr;
+            //}
+
+            //// As per MSDN marking the window as fullscreen should ensure that the
+            //// taskbar is moved to the bottom of the Z-order when the fullscreen window
+            //// is activated. If the window is not fullscreen, the Shell falls back to
+            //// heuristics to determine how the window should be treated, which means
+            //// that it could still consider the window as fullscreen. :(
+            //if (task_bar_list_)
+            //    task_bar_list_->MarkFullscreenWindow(hwnd_, !!fullscreen);
+        }
+
+        private void SetFullScreen(bool fullscreen)
+        {
+            // Ported from https://github.com/chromium/chromium/blob/master/ui/views/win/fullscreen_handler.cc
+            //std::unique_ptr<ScopedFullscreenVisibility> visibility;
+
+            // With Aero enabled disabling the visibility causes the window to disappear
+            // for several frames, which looks worse than doing other updates
+            // non-atomically.
+            //if (!ui::win::IsAeroGlassEnabled())
+            //  visibility = std::make_unique<ScopedFullscreenVisibility>(hwnd_);
+
+            // Save current window state if not already fullscreen.
+            if (!_fullScreen)
```

Comment:
After refactors in this class style management was moved to `UpdateWindowProperties` to keep styling code together.

Reply:
Done.